packages/amplify_analytics_pinpoint/ios/amplify_analytics_pinpoint.podspec

```diff
@@ -15,8 +15,8 @@ This code is the iOS part of the Amplify Flutter Pinpoint Analytics Plugin.  The
   s.source           = { :path => '.' }
   s.source_files = 'Classes/**/*'
   s.dependency 'Flutter'
-  s.dependency 'Amplify', '~> 1.10.0'
-  s.dependency 'AmplifyPlugins/AWSPinpointAnalyticsPlugin', '~> 1.10.0'
+  s.dependency 'Amplify', '~> 1.11.0'
```

Comment:
Why is this change in this PR when similar change in other PR https://github.com/aws-amplify/amplify-flutter/pull/665/files ? I suppose it would not matter if other PR merged first, but making sure I'm not missing something.

Reply:
I had branched off my other change... the branch is now updated so this is resolved.